department-of-veterans-affairs/caseflow
# Jonathan/appeals 4279 create new appeal stream form

Resolves [APPEALS-4279](https://vajira.max.gov/browse/APPEALS-4279)

### Description
Created a new component for the "Create new appeal stream" portion of the Split Appeal workflow.

### Acceptance Criteria
- [ ] The user is directed to the create new appeal stream form when a COB or SSC user selects the "Split appeal" option on the Edit Issues screen. 


### Testing Plan
1. As an Intake user, go into Mail Intake, intake a new 10182 and add at least 2 issues.
2. Switch to one of the following users: (DOANBVAN, GOSNEJVACO, SANFORDBVAM, SAIKHBVAJ) that have both COB and CaseReview permissions.
4. Switch over to Caseflow Queue and search for the appeal you just created.
5. Navigate to the Case Details page and look for the "Correct issues" link  in the "Issues" section of the page.
6. Click on the "Correct issues" link, you should now be on the Edit Issues page.
7. You should see a "Split Appeal" button at the bottom on the screen. Click it and you should be redirected to the new component. 
8. The new page should still be within the scope of Caseflow Intake. 
9. Verify that the "Create new appeal stream" section of the progress bar is highlighted, the dropdown works (including the "other" selection) and the checkboxes work.


### User Facing Changes
![image](https://user-images.githubusercontent.com/98970951/182396709-2f0d94d9-1a1b-4710-be2e-bc4dbb00d91b.png)


### Code Documentation Updates
